### ISSUE TITLE
Shouldn't perf-addon only be present in DEV mode?

### ISSUE THREAD

**Issue by jeremenichelli** [Jam3/generator-jam3#291]
If I'm not wrong we are including this even on production which increases the bundle size 👉 https://github.com/Jam3/generator-jam3/blob/master/templates/react/src/framework/index.js#L3

**Comment by BrendanNeufeld** [comment]
I noticed this as well. Maybe scope in a node env variable. 
https://github.com/Jam3/generator-jam3/blob/master/templates/react/src/framework/index.js#L19

**Comment by jeremenichelli** [comment]
Yeah it would be just simply doing:

```
if (process.env.NODE_ENV === 'development') {
  window.Perf = require('react-addons-perf');
}
```

**Comment by njam3** [comment]
Good catch, I'll group this in the dev tools check.

**Comment by njam3** [comment]
Fixed as of nyg-jam3@1.13.3
